packages/webdriverio/src/commands/element/custom$.ts

```diff
@@ -60,7 +60,7 @@ async function custom$ (
     }
 
     if (res && typeof res[ELEMENT_KEY] === 'string') {
-        return await getElement.call(this, strategy, res)
+        return await getElement.call(this, strategy as any, res)
```

Comment:
`getElement` expects a very specific type `Selector`, which is not compatible with the `strategy`. Not sure how to fix this properly, I went with `any` for now